fl4health/clients/basic_client.py

```diff
@@ -201,6 +384,16 @@ def val_step(self, input: torch.Tensor, target: torch.Tensor) -> Tuple[Losses, D
     def train_by_epochs(
         self, epochs: int, current_round: Optional[int] = None
     ) -> Tuple[Dict[str, float], Dict[str, Scalar]]:
+        """
+        Train locally for the specified number of epochs.
+
+        Args:
+            epochs (int): The number of epochs for local training.
+            current_round (Optional[int]): The current FL round.
+
+        Returns:
+            Tuple[Dict[str, float], Dict[str, Scalar]]: The loss and metrics dictionary from the local training.
```

Comment:
Maybe mention that the loss here is a dictionary potentially corresponding to different components of the loss?

Reply:
Similarly for `train_by_steps` maybe